### PR TITLE
E2E Appium - Import wallet and create wallet

### PR DESCRIPTION
**Pull request by Fatxx**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

Adds Webdriver.io support for iOS and Android, check [notion page](https://www.notion.so/E2E-Testing-361fee8ff65048f0b18d6d3dddc5ad9c) for instructions.

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/Views/AccountBackupStep1/index.js [hunk 1]
```diff
@@ -217,7 +219,10 @@ const AccountBackupStep1 = (props) => {
         style={styles.mainWrapper}
         testID={'account-backup-step-1-screen'}
       >
-        <View style={styles.wrapper} testID={'protect-your-account-screen'}>
+        <View
+          style={styles.wrapper}
+          {...generateTestId(Platform, 'protect-your-account-screen')}
```
Comment (sethkfman) [inline app/components/Views/AccountBackupStep1/index.js:224]: Should these be constants like the other files?

File: app/components/UI/SkipAccountSecurityModal/index.js [hunk 2]
```diff
@@ -72,6 +78,8 @@ const SkipAccountSecurityModal = ({
 
   return (
     <ActionModal
+      cancelTestID={'Cancel-Skip'}
```
Comment (sethkfman) [inline app/components/UI/SkipAccountSecurityModal/index.js:81]: Should these be constants?

File: wdio/config/android.config.browserstack.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+config.capabilities = [
+  {
+    platformName: 'Android',
+    noReset: true,
+    fullReset: false,
+    maxInstances: 1,
+    deviceName: 'Android 11 - Pixel 4a API 30',
+    platformVersion: '11',
+    app: '../android/app/build/outputs/apk/debug/app-debug.apk',
+    // appPackage: 'io.metamask',
```
Comment (sethkfman) [inline wdio/config/android.config.browserstack.js:14]: if these lines are not needed remove

File: app/components/Views/AccountBackupStep1/index.js (hunk 1 above)
Comment (cortisiko) [reply app/components/Views/AccountBackupStep1/index.js]: Done.

File: wdio/config/android.config.browserstack.js (hunk 3 above)
Comment (cortisiko) [reply wdio/config/android.config.browserstack.js]: Done

Comment (cortisiko) [reply wdio/config/android.config.browserstack.js]: done

**Review by sethkfman — APPROVED** [review]
LGTM

**Comment by cortisiko** [comment]
@SocketSecurity ignore bufferutil@4.0.7 canvas@2.10.1 utf-8-validate@5.0.10 chromedriver@99.0.0 appium-chromedriver@4.28.0

**Review by cortisiko — APPROVED** [review]

**Comment by cortisiko** [comment]
@SocketSecurity ignore es5-ext@0.10.62 uglify-es@3.3.9 uglify-js@3.17.4

**Pull request merged**